quickannotator/db/models.py

```diff
@@ -136,7 +137,7 @@ class Notification(Base):
     message_type = Column(Integer, nullable=False)
     is_read = Column(Boolean, nullable=False)
     message = Column(Text, nullable=False)
-    datetime = Column(DateTime, server_default=func.now())
+    datetime = Column(DateTime)
```

Comment:
i guess this was the reason for using now()? tit was possible to set the default? i guess one can't use e.g., datetime.now as the default?

Reply:
Yes. I guess we can still use func now in model schema where we don't need e.g., bulk insertion, but I don't personally see the disadvantage of using serverside `datetime.now()`. Do you have anything in mind?

Reply:
1. sounds right
2. not sure - it should default into the correct datetime, so it may work out? probably worth either testing.....or just set explicitly ;-)


what do you think?

Reply:
Since bulk insertions bypass the ORM, they won't use default values specified in the model schema.

Anyways, I've gone and set the default to datetime.now in the respective models, and removed the explicit datetime.now() calls from e.g., add_project()